yb-voyager/cmd/export.go

```diff
@@ -69,6 +69,9 @@ func init() {
 
 	exportCmd.Flags().BoolVar(&disablePb, "disable-pb", false,
 		"true - to disable progress bar during data export (default false)")
+
+	exportCmd.Flags().StringVar(&source.ExcludeTableList, "exclude-table-list", "",
```

Comment:
Instead of `no-op` or silently ignoring. Please add a check in somewhere `PreRun` in case both the table-list flags are provided then we should error out with a message to user, since only of them makes more sense at a time.